doc/sphinx-guides/source/installation/external-tools.rst

```diff
@@ -17,6 +17,8 @@ Support for external tools is just getting off the ground but the following tool
 
 - `Whole Tale <https://wholetale.org>`_: a platform for the creation of reproducible research packages that allows users to launch containerized interactive analysis environments based on popular tools such as Jupyter and RStudio. Using this integration, Dataverse users can launch Jupyter and RStudio environments to analyze published datasets. For more information, see the `Whole Tale User Guide <https://wholetale.readthedocs.io/en/stable/users_guide/integration.html>`_.
 
+- `File Previewers <https://github.com/QualitativeDataRepository/dataverse-previewers>`_: A set of tools that display the content of files - including audio, html, `Hypothes.is <https://hypothes.is/>` annotations, images, PDF, text, video - allowing them to be viewed without downloading. The previewers can be run directly from github.io, so the only required step is using the Dataverse API to register the ones you want to use. Documentation, including how to optionally brand the previewers, and an invitation to contribute through github are in the README.md file. 
```

Comment:
First, the previewers are awesome. Thanks! My only comment is that while it doesn't look like a download, the download count is incremented when you explore. You **could** say something like "all within your browser" if you want but I'm approving this. Thanks!

Reply:
That's odd - the previewers  use ?gbrecs=false when pulling content for the preview and only use ?gbrecs=true if/when you hit the download button. Is that not the right way to do it?

Reply:
Which is exactly backwards - gbrecs=true suppresses counting the download - I've flipped these in the previewers so this now works correctly...

Reply:
@qqmyers oh! Thanks!